fine_print.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*'] + ['MIT-LICENSE', 'Rakefile', 'README.md']
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'rails', '>= 3.1'
+  s.add_dependency 'rails', '>= 3.1', '< 5.0'
```

Comment:
What's the reasoning for restricting Rails to < 5?

Reply:
@Dantemss I have just answered my own question by looking at the commit messages... my question may be ignored. *However*!!! If you look at the PR I submitted (https://github.com/lml/fine_print/pull/24), I migrated away from Squeel to BabySqueel, which is supported in Rails > 5, so the Rails restriction can be removed if my PR is merged. Thoughts?

Reply:
I will rebase and merge after merging yours.